Content.Shared/Gatherable/GatherableByHandDoAfterEvent.cs

```diff
@@ -0,0 +1,10 @@
+using Content.Shared.DoAfter;
+using Robust.Shared.Map;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.Spaceshroom;
+
+[Serializable, NetSerializable]
+public sealed class GatherableByHandDoAfterEvent : SimpleDoAfterEvent
```

Comment:
FYI you only need to mark networked data as netserializable otherwise it's a waste (in this case it should be marked as such).